lib/schema_validations/active_record/validations.rb

```diff
@@ -202,7 +204,7 @@ def _filter_validation(macro, name) #:nodoc:
           types << match[1].to_sym
         end
         return false if config.only        and not Array.wrap(config.only).include?(name)
-        return false if config.except      and     Array.wrap(config.except).include?(name)
+        return false if config.whitelist      and     Array.wrap(config.whitelist).include?(name)
         return false if config.only_type   and not (Array.wrap(config.only_type) & types).any?
         return false if config.except_type and     (Array.wrap(config.except_type) & types).any?
```

Comment:
> Do you think it's better to change except_type to whitelist_type as well? Just to be consistent.

Yeah makes sense.  Though I really have no idea if anybody uses the `*_type` options in practice.

I think though that the logic should be slightly different than what you have.  I think a whitelist's behavior should be "no matter what other options may be given, whitelisted columns don't get validations added" (which is subtly different from `except` because of interactions with the other options).  So I think the `schema_validations` method doesn't need to change, and the `_filter_validation` logic should be

```ruby
return false if config.whitelist and ...
return false if config.whitelist_type and ...
return false if config.only and ...
return false if config.except and...
return false if config.only_type and ...
return false if config.except_type and...
```

what do you think?